ksqldb-serde/src/main/java/io/confluent/ksql/serde/delimited/KsqlDelimitedDeserializer.java

```diff
@@ -145,6 +148,16 @@ private static Parser timestampParser(final SqlType sqlType) {
     return v -> new Timestamp(Long.parseLong(v));
   }
 
+  private static Parser toBytes(final SqlType sqlType) {
+    return v -> {
+      try {
+        return ByteBuffer.wrap(Base64.getMimeDecoder().decode(v));
```

Comment:
Updated to assign the encoder/decoder to a static constant. Lazy encoding would be pretty difficult to implement since a lot of things (functions, comparisons) depend on BYTES being a ByteBuffer object.